packages/testing/config/storybook/main.js

```diff
@@ -51,7 +54,7 @@ const baseConfig = {
     // ** PLUGINS **
     sbConfig.plugins = [
       ...sbConfig.plugins,
-      ...getSharedPlugins(isEnvProduction),
+      // ...getSharedPlugins(isEnvProduction),
```

Comment:
@peterp I could use some help debugging this one. I get an "undefined" error when trying to run or build storybook if this is enabled.

Reply:
Scratch that. After re-linking my project, I can no longer get Storybook to build or run.